compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -598,7 +598,7 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
 
     def assocs: List[(Name, Tree)] = assocsFromApply(a.tree)
 
-    def symbol: Symbol = a.tree.symbol
+    def symbol: Symbol = a.symbol
```

Comment:
Is this even necessary at all? Looks like it could be removed.

Reply:
Indeed, this should not be needed anymore now that the backendinterface has been removed.